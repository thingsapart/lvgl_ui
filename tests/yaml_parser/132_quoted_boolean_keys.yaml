# In JSON, all keys must be strings. This tests that keys that
# look like booleans are correctly parsed as strings when quoted.
{ "true": "is a string key", "false": 1, ok: true }
