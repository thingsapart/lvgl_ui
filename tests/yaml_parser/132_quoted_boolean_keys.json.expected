[{
		"true":	"is a string key",
		"false":	1,
		"ok":	true
	}]
